### DISCUSSION TITLE
simulate Frequency Division Duplex (FDD) 5G uplink and downlink

### DISCUSSION THREAD

**Discussion by anasash11** [NVlabs/sionna#502] [General]
Dears, 
How can I simulate Frequency Division Duplex (FDD) 5G uplink and downlink using different carrier frequencies with Sionna? Please provide a detailed explanation of the steps involved, including how to configure the carrier frequencies for both uplink and downlink. Additionally, explain the process of finding and analyzing partial channel reciprocity between the uplink and downlink channels. What are the key parameters to consider, and how can they be measured and compared effectively?

**Comment by SebastianCa** [comment]
Hi @anasash11,
please understand that we cannot provide such a detailed support. However, please have a look at the [5G NR PUSCH Tutorial](https://nvlabs.github.io/sionna/examples/5G_NR_PUSCH.html) and the [MIMO OFDM Tutorial](https://nvlabs.github.io/sionna/examples/MIMO_OFDM_Transmissions_over_CDL.html) for FDD uplink/downlink simulations.

**Comment by KanavSabharwal** [comment]
Hi @anasash11,

Thanks for raising this discussion. I came across it while exploring a very similar simulation myself. I realize this was posted a while ago, but I was wondering if you happened to make any progress or discoveries since then? If you have any leads, references, open implementations, or even general suggestions, I'd greatly appreciate it if you could share them.

Looking forward to hearing from you, and thanks in advance!
